src/components/KFileUpload/KFileUpload.vue

```diff
@@ -285,22 +286,38 @@ export default defineComponent({
 .k-file-upload {
   position: relative;
 
+  :deep(.k-input-label) {
+    pointer-events: none;
```

Comment:
question: I don't think we want to block pointer-events on the label, is there a reason this is needed?

Reply:
The label can be clickable (this is standard web behavior) I think the issue was just the `.help` container was also clickable. 

I'd suggest removing the pointer-events rules you added, and then I think everything else LGTM

Reply:
Updated.